### PR TITLE
Som 2

### PR DESCRIPTION
**Pull request by LaugeSolvang**

### CODE REVIEW

**Review by MGertz — CHANGES_REQUESTED** [review]
Be aware some files isnt going into the correct folders.
Make sure to use folder sortingmodule/scoring/...

File: src/main/java/dk/sdu/se_f22/som2/IScoring.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package dk.sdu.se_f22.som2;
```
Comment (The0mikkel) [inline src/main/java/dk/sdu/se_f22/som2/IScoring.java:1]: This file should not be located here?!
Please insert all of your codes into sortingmodule/scoring

File: src/main/java/dk/sdu/se_f22/som2/Scoring.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package dk.sdu.se_f22.som2;
```
Comment (The0mikkel) [inline src/main/java/dk/sdu/se_f22/som2/Scoring.java:1]: The same is applicable here

File: src/main/java/dk/sdu/se_f22/som2/Test.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package dk.sdu.se_f22.sortingmodule.scoring;
+package dk.sdu.se_f22.som2;
```
Comment (The0mikkel) [inline src/main/java/dk/sdu/se_f22/som2/Test.java:1]: The same is applicable here.

**Review by The0mikkel — CHANGES_REQUESTED** [review]
Please move all of your code into `sortingmodule/scoring/...`, so that any code is present, in the correct location and not floating around.

File: src/main/java/dk/sdu/se_f22/sortingmodule/scoring/Scoring.java [hunk 4]
```diff
@@ -1,88 +1,278 @@
...
+        if (instance == null) {
+            instance = new Scoring();
+        }
+        return instance;
+    }
+
+    private void initializePostgresqlDatabase() {
+        try {
+            DriverManager.registerDriver(new org.postgresql.Driver());
+            connection = DriverManager.getConnection("jdbc:postgresql://" + url + ":" + port + "/" + databaseName, username, password);
```
Comment (The0mikkel) [inline src/main/java/dk/sdu/se_f22/sortingmodule/scoring/Scoring.java:30]: I would like you to use the common database connection, with is provided in the "DBConnection" -> `DBConnection.getPooledConnection();`
Then follow the readme in the main repository branch, where it says on how to configure this connection (through "config.properties"

**Comment by The0mikkel** [comment]
I'm not quite sure, how we, as the infrastructure in the SOM module, should use your system?
Is ant of this code made for us to use, or is it part of the crud?

We would very much just like to send the whole "SearchHits" object to you guys, where from here, you just sort in that, or how was it though on your side?

Please see my commect regarding database connection. With your current setup, we would not be able to use your code.

File: src/main/java/dk/sdu/se_f22/sortingmodule/scoring/Scoring.java (hunk 4 above)
Comment (The0mikkel) [inline src/main/java/dk/sdu/se_f22/sortingmodule/scoring/Scoring.java:30]: I would like you to use the common database connection, with is provided in the "DBConnection" -> `DBConnection.getPooledConnection();`
Then follow the readme in the main repository branch, where it says on how to configure this connection (through "config.properties"

**Review by The0mikkel — CHANGES_REQUESTED** [review]

**Pull request merged**